website/docs/best-practices/how-we-mesh/mesh-3-structures.md

```diff
@@ -42,15 +48,37 @@ Horizontal splits separate your DAG based on source or domain. These splits are
 - **Data from different sources.** For example, clickstream event data and transactional ecommerce data may need to be modeled independently of each other.
 - **Team workflows.** For example, if two embedded groups operate at different paces, you may want to split the projects up so they can move independently.
 
+
+<Lightbox src="/img/best-practices/how-we-mesh/horizontal_split.png" title="A simplified dbt DAG with a dotted line representing a horizontal split." />
+
 ### Combining these strategies
 
 - **These are not either/or techniques**. You should consider both types of splits, and combine them in any way that makes sense for your organization.
 - **Pick one type of split and focus on that first**. If you have a hub-and-spoke team topology for example, handle breaking out the central platform project before you split the remainder into domains. Then if you need to break those domains up horizontally you can focus on that after the fact.
 - **DRY applies to underlying data, not just code.** Regardless of your strategy, you should not be sourcing the same rows and columns into multiple nodes. When working within a mesh pattern it becomes increasingly important that we don't duplicate logic or data.
 
+
+<Lightbox src="/img/best-practices/how-we-mesh/combined_splits.png" title="A simplified dbt DAG with two dotted lines representing both a vertical and horizontal split." />
+
+
 ## Determine your git strategy
 
 A multi-project architecture can exist in a single repo (monorepo) or as multiple projects, with each one being in their own repository (multi-repo).
 
 - If you're a **smaller team** looking primarily to speed up and simplify development, a **monorepo** is likely the right choice, but can become unwieldy as the number of projects, models and contributors grow.
 - If you’re a **larger team with multiple groups**, and need to decouple projects for security and enablement of different development styles and rhythms, a **multi-repo setup** is your best bet.
+
+## Projects, splits, and teams
+
+Since the laumch of dbt Mesh, the most common pattern we've seen is one where **projects are 1:1 aligned to teams**, and each project is its own codebase in its own repository. This isn’t a hard-and-fast rule: Some organizations want multiple teams working out of a monorepo. Some teams own multiple domains that feel awkward to keep combined.
```

Comment:
```suggestion
Since the launch of dbt Mesh, the most common pattern we've seen is one where projects are 1:1 aligned to teams, and each project has its own codebase in its own repository. This isn’t a hard-and-fast rule: Some organizations want multiple teams working out of a single repo, and some teams own multiple domains that feel awkward to keep combined.
```